xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountNftsRequestParams.java

```diff
@@ -0,0 +1,51 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.primitives.UnsignedInteger;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.model.transactions.Marker;
+
+import java.util.Optional;
+
+/**
+ * Request parameters for the "account_ntfs" rippled API method.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableAccountNftsRequestParams.class)
+@JsonDeserialize(as = ImmutableAccountNftsRequestParams.class)
+public interface AccountNftsRequestParams {
```

Comment:
Where do the rippled method params/result definitions live? I don't see any in the xls-20d spec

Reply:
https://github.com/XRPLF/XRPL-Standards/discussions/46

Reply:
There is no concrete documentation as of now. Greg put up his binary-codec and NFT spec PR as per his understanding from the  xls-20d spec and his conversations with Nik about it. And this might change, hence they should be left in PR.

Reply:
The current code matches the binary-codec as [specified in the JS SDK here](https://github.com/XRPLF/xrpl.js/pull/1829/files#diff-3bfe85079db79c97868c6e30ff908327bb3cdb70be62cc01a7c46279a169d0a2).